### DISCUSSION TITLE
Don't re-sort on edit / Only sort when clicking header

### DISCUSSION THREAD

**Discussion by JRebella** [TanStack/table#3145] [Q&A]
Hi, I'm trying to build a table with multiple editable fields and sorting. The issue is that when the table is sorted by column X, and user edits value X in some row, the row changes position and will confuse the user

Is there a way to ONLY sort the rows when the user clicks on a column header? If user edits a field, just keep the table out of order until they click a column header to sort again

Thanks!

**Comment by AlexPosto** [comment]
@JRebella did you ever found a solution to this ?
